### PR TITLE
Add somre more folders to eslint import order rules

### PR DESCRIPTION
**Pull request by lfryemason**
In order to improve code cleanliness slightly, we are rolling out an eslint rule to the entire repo that ensure .js, .jsx, .ts and .tsx files have imports automatically sorted. 

We are rolling this out incrementally to prevent errors and decrease PR size. This PR removes a subset of folders from the rule exclusions.

FLUP of [#57247](https://github.com/code-dot-org/code-dot-org/pull/57247)

**For PR reviewers**: please double check that all comments in imports are appropriately moved with the line they are talking about.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by Hamms — COMMENTED** [review]
Do the circular dependency errors you were seeing represent actual new circular dependencies, or are they just the result of the existing circular dependencies showing up in a different order? See https://github.com/code-dot-org/code-dot-org/pull/58887#issuecomment-2135327537 for an example

**Comment by lfryemason** [comment]
> Do the circular dependency errors you were seeing represent actual new circular dependencies, or are they just the result of the existing circular dependencies showing up in a different order? See [#58887 (comment)](https://github.com/code-dot-org/code-dot-org/pull/58887#issuecomment-2135327537) for an example

They are because the order of imports change. I assume this is because the existing circular dependencies don't cause problems when they are imported in a specific order, but this would take time to confirm.

**Comment by Hamms** [comment]
I think it's worth taking that time! Updating the circular dependencies tracker at `apps/circular_dependencies.json` should be pretty straightforward

**Comment by lfryemason** [comment]
> circular_dependencies

I didn't know about this `circular_dependencies.json` list. I removed the other places I disabled the import rule and also added some clarifying text to the output so it is easier to find for the next person.

File: apps/webpack.config.js
Comment (Hamms) [inline apps/webpack.config.js:112]: :+1:

**Review by Hamms — APPROVED** [review]
Yeah, this circular dependencies functionality is relatively new. Thanks for taking the time to make it more discoverable!

This changeset LGTM, once that merge conflict gets resolved. Thank you!

**Pull request merged**
